### PR TITLE
Trustly: NotifyParent

### PR DESCRIPTION
**Pull request by palmenhq**
This should redirect members to bankid if needed from trustly's side. I'm unsure what will happen after that though, as i've been unable to test it manually :|

### CODE REVIEW

**Comment by gustaveen** [comment]
Good job! 🎉  Do we have any way to test Trustly manually?

File: src/client/pages/ConnectPayment/components/TrustlyModal.test.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+    expect(wrapper.find('Header').text()).toBe(
+      'ONBOARDING_CONNECT_DD_TRUSTLY_MODAL_TITLE',
+    )
+    expect(wrapper.find('iframe').prop('src')).toBe(trustlyUrl)
+  })
+
+  it('redirects to success page when iframe is successful', async () => {
+    const trustlyUrl = 'http://trustly.com/blah'
+    const ShowPath = withRouter(({ history }) => {
```
Comment (gustaveen) [inline src/client/pages/ConnectPayment/components/TrustlyModal.test.tsx:81]: Did you find any explanation to why we couldn't use the hook? :D

Comment (palmenhq) [reply src/client/pages/ConnectPayment/components/TrustlyModal.test.tsx:81]: Not 100% but there seems to be an issue with enzyme + `useContext` in some way. I spent 3 hours on trying to get it to work unsuccessfully and then I figured hooks are cool but not that cool 😅

**Comment by palmenhq** [comment]
@gustaveen not RN, I’m waiting for their tech support to get back to me :)

**Comment by gustaveen** [comment]
> @gustaveen not RN, I’m waiting for their tech support to get back to me :)

Cool! Would just feel nice to test this before we release it to production :) Otherwise it looks all good to me 👍

**Review by gustaveen — APPROVED** [review]
Let's merge! 🥳

**Pull request merged**